### ISSUE TITLE
Download via torrent

### ISSUE THREAD

**Issue by douglasanpa** [popcorn-time/popcorn-time.github.io#37]
the system is in torrent, why not put the download on torrent and the mega.co.nz if the mega.co.nz fails can still download via torrent

**Comment by tiagoad** [comment]
I suggest dropping mega and switching to github releases.

**Comment by Shrinkingcash** [comment]
I fully agree about the torrent!!

**Comment by miguelfrde** [comment]
Why not both options? Direct download via github releases and torrent download?

**Comment by remil19** [comment]
If the idea is to bring torrents to new people; keep  the direct link in evidence and add a discreet torrent magnet : people who are already used to torrents are used to be looking for such things but if someone click on the magnet link by mistake, he will certainly drop the idea.

**Comment by krisu5** [comment]
Both direct download and torrent, but torrent should be always secondary way.

**Comment by remil19** [comment]
I don't think so. There should be a direct download link in big and a small text link for torrents, for user who already use it but are happy to have a new user-friendly software. Moreover it could be very useful if the main site is forced to close.
